src/client/liveblog.ts

```diff
@@ -1,45 +1,50 @@
 // ----- Imports ----- //
 
-import setup from 'client/setup';
-import { fromCapiLiveBlog, getFormat } from 'item';
 import ReactDOM from 'react-dom';
-import LiveblogBody from 'components/liveblog/body';
 import { createElement as h } from 'react';
-import { Content } from 'mapiThriftModels';
-import { parse } from './parser';
+import { Format, Design, Display, Pillar } from '@guardian/types/Format';
 
+import setup from 'client/setup';
+import { fromSerialisable } from 'liveBlock';
+import { parse } from 'client/parser';
+import LiveblogBody from 'components/liveblog/body';
 
-// ----- Run ----- //
 
-declare global {
-    interface Window {
-        __INITIAL__DATA__: Content;
-    }
+// ----- Setup ----- //
+
+const domParser = new DOMParser();
+
+const format: Format = {
+    design: Design.Live,
+    display: Display.Standard,
+    pillar: Pillar.News,
 }
 
-setup();
 
-const browserParser = (html: string): DocumentFragment =>
-    parse(new DOMParser())(html)
+// ----- Functions ----- //
+
+const docParser = (html: string): DocumentFragment =>
+    parse(domParser)(html)
         .toOption()
-        .withDefault(new DocumentFragment())
+        .withDefault(new DocumentFragment());
+
+const deserialise = fromSerialisable(docParser);
 
 
-try {
-    const hydrationProps = JSON.parse(document.getElementById('hydrationProps')?.textContent ?? "");
+// ----- Run ----- //
+
+// Set up interactives, Twitter embeds etc.
+setup();
 
-    if (hydrationProps) {
-        const item = fromCapiLiveBlog({
-            docParser: browserParser,
-            salt: 'mockSalt',
-        })(hydrationProps);
-        const { blocks, totalBodyBlocks } = item;
+// Load the initial group of blocks.
+fetch(`${window.location}/live-blocks`)
```

Comment:
Can you use the relative path? 

`fetch(`/live-blocks`)`

Reply:
That will request `localhost:8080/live-blocks` I'm afraid.

Reply:
Oh so you need the id passed in the request?

Reply:
Well we need some way of identifying the article so we can send a request to CAPI for the blocks. We *could* do that by pulling it out of the "referer" header I suppose...